pkg/spdk/spdk.go

```diff
@@ -0,0 +1,724 @@
+package spdk
```

Comment:
I will skip the review of this commit since we will use the implementation in `longhorn-spdk-engine` in the end.